superbench/cli/_result_handler.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""SuperBench CLI result subgroup command handler."""
+
+from knack.util import CLIError
+
+from superbench.analyzer import DataDiagnosis
+from superbench.common.utils import create_sb_output_dir
+from superbench.cli._handler import check_argument_file
+
+
+def diagnosis_command_handler(raw_data_file, rule_file, baseline_file, output_dir=None, output_file_format='excel'):
+    """Run data diagnosis.
+
+    Args:
+        raw_data_file (str): Path to raw data jsonl file.
+        rule_file (str): Path to baseline yaml file.
+        baseline_file (str): Path to baseline json file.
+        output_dir (str): Path to output directory.
+        output_file_format (str): Format of the output file, 'excel' or 'json'. Defaults to 'excel'.
+    """
+    try:
+        # Create output directory
+        sb_output_dir = create_sb_output_dir(output_dir)
```

Comment:
What happens if the folder is already created?

Reply:
the new ouput file will overwrite the original file if it exists, because the output file name is hardcode, other files in the folder are not affected